### PR TITLE
test: add drag and drop test cases

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes https://github.com/cheminfo/nmrium/issues/1755

### CODE REVIEW

**Comment by wadjih-bencheikh18** [comment]
@lpatiny dropping .nmrium files throw an error

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "eventemitter3": "^4.0.7",
     "file-saver": "^2.0.5",
     "formik": "^2.2.9",
+    "fs": "^0.0.1-security",
```
Comment (lpatiny) [inline package.json:61]: Very strange import that should be removed.

**Review by lpatiny — COMMENTED** [review]
Great many thanks !

Since yesterday ... we are also able to drag / drop a full folder of files (and not only a zip file).

I just added a folder test-e2e/data/bruker

Please could you also add a test that simulate the drag / drop of this full folder ?

The tab 1H should be there that contains 2 entries

image

**Comment by targos** [comment]
@lpatiny Did you see https://github.com/cheminfo/nmrium/pull/1770#issuecomment-1256977717 ?

**Comment by lpatiny** [comment]
> @lpatiny Did you see [#1770 (comment)](https://github.com/cheminfo/nmrium/pull/1770#issuecomment-1256977717) ?

@wadjih-bencheikh18 Is it still the case ? Because all the tests are passing and trying locally works correctly. Maybe it was solved when we merge nmr-load-save branch.

**Comment by wadjih-bencheikh18** [comment]
> Is it still the case ?

no it's fixed

**Pull request merged**